doc/classes/CanvasItem.xml

```diff
@@ -647,17 +647,17 @@
 		</signal>
 		<signal name="hidden">
 			<description>
-				Emitted when becoming hidden.
+				Emitted when the [CanvasItem] becomes hidden, either because its own [member visible] property was set to [code]false[/code] or the [member visible] property of one of its [CanvasItem] ancestors was set to [code]false[/code].
 			</description>
 		</signal>
 		<signal name="item_rect_changed">
 			<description>
-				Emitted when the item's [Rect2] boundaries (position or size) have changed, or when an action is taking place that may have impacted these boundaries (e.g. changing [member Sprite2D.texture]).
+				Emitted when the [CanvasItem]'s boundaries (position or size) have changed, or when an action is taking place that may have impacted these boundaries (e.g. changing [member Sprite2D.texture]).
```

Comment:
```suggestion
				Emitted when the [CanvasItem]'s boundaries (position or size) changes, or when an action is taking place that may have impacted these boundaries (e.g. changing [member Sprite2D.texture]).
```
Let's match the tense for these

Reply:
There's a general inconsistent use of past tense for signal descriptions. I'd say using the present consistently is fine.

Reply:
Here the consistency is local as all the other signals here use it

Reply:
Meant fixing this method that's already being changed, it's just this one and we're already editing it, so why not go all the way

Reply:
I think I fixed this with a solution somewhere in the middle of both of your suggestions!